pages/28.Marketplace/default.en.md

```diff
@@ -0,0 +1,138 @@
+
+The license should be GPL v3 mostly as Mautic is released under GPL v3 which is a viral license. Which means whatever is using Mautic's code base should also be released under the same license.
+
+Required packages are dependencies. The bigger is the list of dependencies the bigger will be the size of the plugin. More dependencies also means more security risks and incompatibility issues with future upgrades.
+
+#### All Versions
+
+The next table shows the list of all versions. *In the future versions of the marketplace it should be possible to select which version you want to install or upgrade to.*
+
+From the list you can see how many times the plugin has been released and what is the release cadence. When you click on a specific version it will open a new window where the plugin maintainers should write change log. Meaning what has been added or which bugs were fixed in the specific version.
+
+#### Maintainers
+
+In this section is a list of maintainers of the plugin on Packagist. There may be more contributors in the GitHub repository. There is also a link to the maintainer's Packagist detail page where you can browse other PHP packages of the same author.
+
+#### GitHub Info
+
+[GitHub](https://github.com) is a social network for developers. The majority of Mautic plugins is available on GitHub. There are some quick stats available directly in this section but you can find much more if you follow the link to the repository.
+
+#### Packagist Info
+
+[Packagist](https://packagist.org) is a PHP package repository. It's not related specifically to the Mautic community but to PHP community. All the PHP packages listed in Packagist are installable by [Composer](https://getcomposer.org) which is a tool for dependency management which is used under the hood when you install a Mautic plugin. Packagist section shows download stats of how many the plugin has been installed in different timeframes.
+
+#### Context menu
+
+The context menu shows actions you can take.
+- Close will take you to the List View
+- Install will install the plugin
+- Issue Tracker will open a new window with the issue tracker. It will show up only if the plugin has this information available. Use this option to search for issues with the pluin and report new ones.
+
+![Marketplace detail](marketplace-detail.png)
+
+### CLI command
+
+The marketplace has commands that can be handy for those who prefer CLI over UI or for some automations.
+
+#### List plugins
+
+`bin/console mautic:marketplace:list` will list first page of available plugins like so:
+
+```
++-------------------------------------------------------+-----------+--------+
+| name                                                  | downloads | favers |
```

Comment:
Github refers to the count of folk having starred a repo as 'stargazers' - favers did not make sense really. Could also call it 'likes' but that does not correlate too well with GH?

```suggestion
| name                                                  | downloads | stargazers |
```

Reply:
favers is the original term used by Packagist. Do we want to rename it?

Reply:
Ah OK - it just doesn't seem to make sense to me, like, it's not really a word. If that is what they call people who have starred a repo I guess we keep it?